docs/source/_templates/indexcontent.html

```diff
@@ -14,29 +14,29 @@ <h1>Hyperledger Sawtooth documentation</h1>
   <p class="biglink"><a class="biglink" href="{{ pathto("introduction") }}">{% trans %}Introduction{% endtrans %}</a><br/>
     <span class="linkdescr">{% trans %}learn about Hyperledger Sawtooth{% endtrans %}</span></p>
 
-  <p class="biglink"><a class="biglink" href="{{ pathto("architecture") }}">{% trans %}Architecture{% endtrans %}</a><br/>
-    <span class="linkdescr">{% trans %}high-level architecture and design topics{% endtrans %}</span></p>
+  <p class="biglink"><a class="biglink" href="{{ pathto("architecture") }}">{% trans %}Architecture Description{% endtrans %}</a><br/>
+    <span class="linkdescr">{% trans %}read high-level architecture and design topics{% endtrans %}</span></p>
 
   <p class="biglink"><a class="biglink" href="{{ pathto("app_developers_guide") }}">{% trans %}Application Developer's Guide{% endtrans %}</a><br/>
     <span class="linkdescr">{% trans %}create applications using the Sawtooth platform{% endtrans %}</span></p>
 
-  <p class="biglink"><a class="biglink" href="{{ pathto("sysadmin_guide") }}">{% trans %}System Administrator's Guide{% endtrans %}</a><br/>
-    <span class="linkdescr">{% trans %}installation, configuration, and maintenance{% endtrans %}</span></p>
+  <p class="biglink"><a class="biglink" href="{{ pathto("transaction_family_specifications") }}">{% trans %}Transaction Family Specifications{% endtrans %}</a><br/>
+    <span class="linkdescr">{% trans %}dive into the design of existing transaction families{% endtrans %}</span></p>
 
-  <p class="biglink"><a class="biglink" href="{{ pathto("rest_api") }}">{% trans %}REST API Reference{% endtrans %}</a><br/>
-    <span class="linkdescr">{% trans %}submit transactions and retrieve block, transaction, and state data{% endtrans %}</span></p>
+  <p class="biglink"><a class="biglink" href="{{ pathto("sysadmin_guide") }}">{% trans %}System Administrator's Guide{% endtrans %}</a><br/>
+    <span class="linkdescr">{% trans %}perform installation, configuration, and maintenance{% endtrans %}</span></p>
 
-  <p class="biglink"><a class="biglink" href="{{ pathto("cli") }}">{% trans %}CLI Reference{% endtrans %}</a><br/>
-    <span class="linkdescr">{% trans %}command-line tool reference{% endtrans %}</span></p>
+  <p class="biglink"><a class="biglink" href="{{ pathto("api_references") }}">{% trans %}API References{% endtrans %}</a><br/>
+    <span class="linkdescr">{% trans %}explore the Sawtooth API reference guides{% endtrans %}</span></p>
 
-  <p class="biglink"><a class="biglink" href="{{ pathto("transaction_family_specifications") }}">{% trans %}Transaction Family Specifications{% endtrans %}</a><br/>
-    <span class="linkdescr">{% trans %}dive into design of existing transaction families{% endtrans %}</span></p>
+  <p class="biglink"><a class="biglink" href="{{ pathto("cli") }}">{% trans %}CLI Command Reference{% endtrans %}</a><br/>
```

Comment:
I suppose this has already been hashed out, and decisions have been made, but it seems odd to me that we have the CLI reference at the top level, but the REST API and SDK references tucked away in an API References section. Are all three not APIs? If not, it seems like the difference is more technicality than material. I would think all three should be top level, or all three in one section (whatever you want to call it).

Just my two cents.

Reply:
I don't think of a CLI as an application programming interface (or, as Wikipedia calls it, an "application protocol interface").  Yes, an app could run Sawtooth commands, but I believe that the main intended use is by humans and shell scripts.  Also, this organization is per @vaporos' recommendation - he wanted to see the CLI reference at the top level. (That was his two cents.)